### PR TITLE
run rpi-jupyter in rootless mode

### PR DESCRIPTION
**Pull request by movalex**
Here's my proposal of making this image running under unprivileged user, like in `jupyter/minimal-notebook`. It is also working fine under Raspbian Jessie (kernel 4.4), so I mentioned it in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,36 +13,52 @@ Due to a popular python library called scikit-learn requires bzip2 library which
 I have also built maxjiang/rpi-jupyter:datascience (based on 1.1) so that you have most of the data science packages installed without installing/compiling them yourself.
 
 ### Installing
-Go to [Hypriot OS](http://blog.hypriot.com/) and follow the steps to get the Raspberry Pi docker ready. Then, run the following:
+Go to [Hypriot OS](https://blog.hypriot.com/downloads) and follow the steps to get this OS. However if you have [Raspbian Jessie](https://www.raspberrypi.org/downloads/raspbian/) OS installed, you are totally fine too, it will work.
```
Comment (mkjiang) [inline README.md:16]: I actually did not realise Docker officially supported Raspberry Pi until now. Mentioning Hypriot OS is not necessary here so please rephrase if you like.

File: README.md [hunk 2]
```diff
@@ -13,36 +13,52 @@ Due to a popular python library called scikit-learn requires bzip2 library which
...
+If you would like to change some config, create your own jupyter_notebook_config.py (or use sample file from this repository) on the docker host and run the following:
+
+    docker run -it -p <host port>:<dest port> -v <path to your config file>:/home/jovyan/.jupyter/jupyter_notebook_config.py maxjiang/rpi-jupyter
 
 This maps a local config file to the container.
 
-The following command gives you a bash session in the running container so you could do more:
+The notebook will run under unpriviledged user `jovyan` (uid=1000) in group `users` with ownership of `/home/jovyan`. If you want to mount your default working directory on the host to preserve work even when notebook is not running or destroyed, use additional `-v` option:
+
+    docker run -it docker run -it -p <host port>:<dest port> -v <path to your config file>:/home/jovyan/.jupyter/jupyter_notebook_config.py -v /some/host/folder/for/work:/home/jovyan/work  maxjiang/rpi-jupyter
```
Comment (mkjiang) [inline README.md:49]: Let's change <path to your config file> to  something like /some/host/folder/jupyter_notebook_config.py to make it consistent.

**Review by mkjiang — COMMENTED** [review]
Thanks man. It's looking good! A couple of comments from me but just nip-picking. Cheers!

File: README.md (hunk 2 above)
Comment (movalex) [reply README.md:49]: Hi! Actually I changed a lot to make it work as I would like to. Maybe it is not completely what you wanted to make, so I decided to pull my own image to Docker. Please have a look on it, if you interested. There's two images, basic and datascience, with a lot of stuff it it. [Basic jupyter conda notebook](https://hub.docker.com/r/movalex/rpi-jupyter-conda/) is rather quick to build, for instance, it has prebuilt tini binary, so you don't have to build it (see [github link](https://github.com/movalex/rpi-jupyter-conda)). [Advanced one](https://github.com/movalex/rpi-jupyter-julia) even has Julia 0.5.1 on board, so you can have a lot of fun making plots on your Raspberry :) Yes, it is slow as hell on my Pi2, but, you now, I did it just for fun.

Comment (mkjiang) [reply README.md:49]: I see. No worries so I guess let's leave it as a separate project then. Will look into Miniconda and Julia. Cheers!

**Pull request closed without merging**